recipe/build-sysroot.sh

```diff
@@ -26,6 +26,11 @@ rm usr/include/rpcsvc/yp_prot.h
 rm usr/include/rpcsvc/ypupd.h
 rm usr/include/rpcsvc/yp.x
 
+# we supply libcrypt/libxcrypt as a separate package
+rm usr/include/crypt.h
+rm lib64/libcrypt*.so*
```

Comment:
I know we removed those for `libsnl` in https://github.com/conda-forge/linux-sysroot-feedstock/pull/40 .
But revisiting this, I think I'd rather have us just remove the `libcrypt.so` below and leave `libcrypt.so.1` (and the symlink target, of course) in place.

Reason is that we use our sysroot packages for `QEMU_LD_PREFIX` and thus will render current builds of packages that link to `libcrypt.so.1` non-functional for emulated builds.
(This is actually not yet too relevant for the 2.12 package since only >=2.17 are used for the platforms we emulate -- but let's make it consistent from the get go.)

Reply:
I'd rather we delete them, so that we know which packages are depending on libcrypt.so.1 and can fix them sooner.

Reply:
I'm about to rebuild `perl` -- that is one of them.
I also know of `ruby`, `python` (well, only the deprecated `crypt` module), `util-linux`.
I'd be more comfortable if we don't break everything at once, i.e., leave `libcrypt*.so*` in place for a couple of months and rebuild the most obvious packages in the meantime.

Reply:
Let's first send PRs to `perl, ruby, python, util-linux` before merging these PR (with or without `libcrypt.so.1`).

Reply:
`perl`, `python`, `ruby`, `util-linux`, `openssh` have now been rebuilt again `libxcrypt`.
I don't know of any others but I also did not really do a structured search for more.

Reply:
Any other comments here, or are we good to keep the code as is and let things break?

Reply:
Personally, I'd still go for the more defensive route -- but if there is a strong opinion on it, I won't oppose to merge as is.

Reply:
I don't have a strong opinion. Can you update these PRs, merge and open an issue to remove `libcrypt.so.1`?

Reply:
I've updated the description of gh-52 and added https://github.com/conda-forge/linux-sysroot-feedstock/issues/52#issuecomment-1945577536 .